mainwindow.cpp

```diff
@@ -19,11 +19,16 @@
 #include <QClipboard>
 #include <QThread>
 #include <QTextCursor>
-#include <QStandardPaths>
+#include <QDesktopServices>
 #include <QInputDialog>
+#include <QtDebug>
 
 #define INTERVAL_PICK_CRACKED 600
-#define PASSWORD_TAB 0
+#define PASSWORD_TAB    0
```

Comment:
It's usually nicer to prefix with the common word. This way you get nice tab completion and it's a bit more evident what they're for.
TAB_PASSWORDS
TAB_OPTIONS
TAB_STATISTICS
....